crates/bevy_ecs/src/system/query.rs

```diff
@@ -15,33 +15,40 @@ use core::{
     ops::{Deref, DerefMut},
 };
 
-/// [System parameter] that provides selective access to the [`Component`] data stored in a [`World`].
+/// A [system parameter] that provides selective access to the [`Component`] data stored in a [`World`].
 ///
-/// Enables access to [entity identifiers] and [components] from a system, without the need to directly access the world.
-/// Its iterators and getter methods return *query items*.
-/// Each query item is a type containing data relative to an entity.
+/// Queries enable systems to access [entity identifiers] and [components] without requiring direct access to the [`World`].
+/// Its iterators and getter methods return *query items*, which are types containing data relative to an entity.
```

Comment:
nit: relative to sounds weird cause its makes me think its brothers with the entity or something. 

Maybe just say "related to" or just "on" an entity